src/app/functions/universal/breach.ts

```diff
@@ -19,3 +19,10 @@ export const BreachDataTypes = {
   HistoricalPasswords: "historical-passwords",
   General: "general",
 } as const;
+
+export const HighRiskDataTypes = {
```

Comment:
Very useful to have one place to reference that lists all the high risk data types :+1: